src/cljs/main/broadfcui/page/workspace/monitor/workflow_details.cljs

```diff
@@ -102,6 +103,21 @@
            (create-field name (display-value value)))
          log-map)])))
 
+(defn- failure-tree [failures]
+  [comps/Tree {:start-collapsed? false
+               :label "Failures"
+               :highlight-ends false
+               :data (prewalk
+                       (fn [elem]
+                         (cond
+                           ;; reorder map elements to message then causedBy
```

Comment:
They're not, but a short map literal will be implemented as a PersistentArrayMap.  This is highly brittle, however.